### PR TITLE
WDP200303-6

### PR DESCRIPTION
**Pull request by ichtys77**
The task was adding oldPrice rendering in ProductBox component.
I:
- use initialState to set values for oldPrice props
- add div with oldPrice button
- add styles for price and oldPrice class

### CODE REVIEW

**Comment by ichtys77** [comment]
its done

File: src/components/common/ProductBox/ProductBox.js [hunk 1]
```diff
@@ -51,6 +50,12 @@ const ProductBox = ({ name, price, promo, stars, image, favorite, compare }) =>
         </Button>
       </div>
       <div className={styles.price}>
+        {oldPrice && <div></div>}
+        {oldPrice && (
```
Comment (Varenthein) [inline src/components/common/ProductBox/ProductBox.js]: What is that?  

```
        {oldPrice && <div></div>}
```

Do we really need this empty div?

Comment (ichtys77) [reply src/components/common/ProductBox/ProductBox.js]: unnecessary artifact, I take it down

**Comment by Varenthein** [comment]
Please resolve conflicts and I think we're good to go :)

**Pull request merged**